interface/main/messages/messages.php

```diff
@@ -556,7 +556,7 @@ function addtolist(sel){
             <td class=\"text\"><a href=\"messages.php?showall=".attr($showall)."&sortby=".attr($sortby)."&sortorder=".attr($sortorder)."&begin=".attr($begin)."&task=addnew&$activity_string_html\" onclick=\"top.restoreSession()\">" .
               htmlspecialchars( xl('Add New'), ENT_NOQUOTES) . "</a> &nbsp; <a href=\"javascript:confirmDeleteSelected()\" onclick=\"top.restoreSession()\">" .
               htmlspecialchars( xl('Delete'), ENT_NOQUOTES) . "</a></td>
-            <td align=right class=\"text\">$prevlink &nbsp; $end of $total &nbsp; $nextlink</td>
+            <td align=right class=\"text amount-msg\">$prevlink &nbsp; $end of $total &nbsp; $nextlink</td>
         </tr>
```

Comment:
I would be happy to discuss possible solutions in a new thread. Should not be awfully complex to implement.

Either printf's formatting "%$1s of %$2s" 
or more user-friendly  "{end} of {total}"  that requires a bit more of processing power.